### PR TITLE
January 2017 Sky Bet Tech Talks post

### PR DESCRIPTION
**Pull request by ls12styler**
Jan '17 Sky Bet Tech Talks post, inclusive of most videos.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -292,3 +292,13 @@ marcus_ojerinde_ardalla:
...
     bio: "A Young Data Engineer With a Passion For All Things Sports"
     github: MarcusDevel
+
+ashley_broadley:
+    name: Ashley Broadley
+    image: ashley_broadley.png
+    role: Principal Engineer
+    tribe: Gaming
+    startDate: 06-02-2012
+    bio: "Likes programming, board games, cycling, Squash, Thai Boxing and Pilates."
```
Comment (WheresAlice) [inline _data/authors.yml]: not sure about the capitalisation here

Comment (ls12styler) [inline _data/authors.yml]: Yes. It is inconsistent. I'll fix it tomorrow.

**Comment by ls12styler** [comment]
@WheresAlice Fixed.

File: _posts/2017-01-24-january-tech-talks.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+First up we have Adele, a Scrum Master looking after two of our Squads in the Gaming Tribe. Adele held a Retrospective Surgery, digging into some issues commonly associated with Retrospectives, prescribing things like fun, games and a safe environment to help get the best out of your Retrospectives.
+(Unfortunately we hit some technical issues with the video for this one, so be sure to come back and read through the blog post version)
+
+### Lean Agile Portfolio Management at Speed!
+Next up, Mark. Mark spoke about the tricks and tools we use in the Gaming Tribe to manage our Portfolio and Programmes. Stuff that keeps everybody, from Execs to Engineers, in the know.
+<iframe width="714" height="402" src="https://www.youtube.com/embed/Bx0WySQWGGQ" frameborder="0" allowfullscreen></iframe>
+
+
+### The Science Behind Innovation
+Tom, Mark and Adam then spoke about how the Sky Vegas Squad, and the wider Gaming Tribe, decide what the 'right' thing to build for our products is, using both user and a/b testing to help validate our ideas.
```
Comment (mitchellrj) [inline _posts/2017-01-24-january-tech-talks.md]: "A/B" should be capitalised

**Pull request merged**